src/category_theory/instances/topological_spaces.lean

```diff
@@ -105,10 +105,19 @@ end Top
 
 variables {X : Top.{u}}
 
-instance : category (opens X) := category_theory.preorder_category (opens X)
+-- To avoid mucky about with opposite categories, we just define the morphisms
+-- in the direction we're always going to want to use.
+instance : category.{u+1} (opens X) :=
+{ hom  := λ U V, ulift (plift (U ≥ V)),
+  id   := λ X, ⟨ ⟨ le_refl X ⟩ ⟩,
+  comp := λ X Y Z f g, ⟨ ⟨ ge_trans f.down.down g.down.down ⟩ ⟩ }
 
-def nbhd (x : X.α) := { U : opens X // x ∈ U }
-def nbhds (x : X.α) : category (nbhd x) := begin unfold nbhd, apply_instance end
+def nbhds (x : X.α) := { U : opens X // x ∈ U }
```

Comment:
Should we call this `open_nhds` to avoid confusion with the `nhds` that is in mathlib?

Reply:
In particular "neighborhood" is abbreviated `nhd` in mathlib, without the "b"--it's hard enough to remember one abbreviation of this word so let's try to be consistent.

Reply:
Okay, I've changed this to `open_nhds`.